wiki/en/Software-Manual.md

```diff
@@ -109,7 +109,12 @@ If you have set your Audio Channel to Stereo or Stereo Out in your Settings, you
 
 If you see a "mute" icon above a user, it means that person cannot hear you. Either they have muted you, soloed one or more users not including you, or have set your fader in their mix to zero.
 
-Users usually appear left-to-right in the order that they connect. You can sort instead by name, instrument, group, or city using the View menu.
+Users usually appear left-to-right in the order that they connect. When joining a server that already has participants, the default order depends
+on the versions of Jamulus at the client and the server. With a client version before 3.12.0 or a server version older than 3.5.5, the existing participants will be shown before
```

Comment:
Will splitting up mid-sentence make translations harder?

Reply:
Don't know, but it wasn't intentional. I can join the lines.